contracts/multisig/src/contract/execute.rs

```diff
@@ -131,9 +131,10 @@ pub fn register_pub_key(
         return Err(ContractError::InvalidPublicKeyRegistrationSignature);
     }
 
-    PUB_KEYS.save(
+    save_pub_key(
         deps.storage,
-        (info.sender.clone(), public_key.key_type()),
+        info.sender.clone(),
+        public_key.key_type(),
         &public_key.clone().into(),
```

Comment:
Why don't we just pass the public key as one parameter, and extract the key_type and call into() within `save_pub_key`?